Content.Server/Shuttles/Systems/ShuttleSystem.Impact.cs

```diff
@@ -54,5 +59,8 @@ private void OnShuttleCollide(EntityUid uid, ShuttleComponent component, ref Sta
         var audioParams = AudioParams.Default.WithVariation(0.05f).WithVolume(volume);
 
         _audio.Play(_shuttleImpactSound, Filter.Pvs(coordinates, rangeMultiplier: 4f, entityMan: EntityManager), coordinates, true, audioParams);
+        
+        var kineticEnergy = ourBody.Mass * Math.Pow(jungleDiff, 2) / 2;
+        _expSys.QueueExplosion(coordinates.ToMap(EntityManager), ExplosionSystem.DefaultExplosionPrototypeId, (float)(kineticEnergy*IntensityMultiplier), 5f, 50f);
```

Comment:
```suggestion
		var mapCoords = coordinates.ToMap(EntityManager)
		var intensity = (float)(kineticEnergy*IntensityMultiplier)
        _expSys.QueueExplosion(mapCoords, ExplosionSystem.DefaultExplosionPrototypeId, intensity , 5f, 50f);
```
Так вроде даже лучше будет, чтобы не растягивать метод на всё лицо

Reply:
ну лан